code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -269,6 +269,29 @@
 	. = ..()
 	AddComponent(/datum/component/armor_plate)
 
+	//Exploration hardsuit
+/obj/item/clothing/head/helmet/space/hardsuit/exploration
+	name = "exploration hardsuit helmet"
+	desc = "An advanced space-proof hardsuit designed to protect against off-station threats."
+	icon_state = "hardsuit0-exploration"
+	item_state = "death_commando_mask"
+	item_color = "exploration"
+	heat_protection = HEAD
+	armor = list("melee" = 35, "bullet" = 15, "laser" = 20, "energy" = 10, "bomb" = 50, "bio" = 100, "rad" = 50, "fire" = 50, "acid" = 75, "stamina" = 20)
```

Comment:
```suggestion
	armor = list("melee" = 35, "bullet" = 15, "laser" = 20, "energy" = 10, "bomb" = 50, "bio" = 70, "rad" = 50, "fire" = 50, "acid" = 75, "stamina" = 20)
```
I see no reason to make them fully disease proof.

Reply:
All hardsuits are fully disease proof.